lib/improver/percentile.py

```diff
@@ -47,8 +47,8 @@ class PercentileConverter(object):
     """
 
     # Default percentile boundaries to calculate at.
-    DEFAULT_PERCENTILES = [0, 5, 10, 20, 25, 30, 40, 50,
-                           60, 70, 75, 80, 90, 95, 100]
+    DEFAULT_PERCENTILES = (0, 5, 10, 20, 25, 30, 40, 50,
+                           60, 70, 75, 80, 90, 95, 100)
 
```

Comment:
Yes, I know, but as #212 has moved the location of this object, there is already going to be a conflict.